src/Datasource/EntityTrait.php

```diff
@@ -382,6 +382,33 @@ protected function isModified(string $field, mixed $value): bool
      * @throws \InvalidArgumentException if an empty field name is passed
      */
     public function &get(string $field): mixed
+    {
+        return $this->getEx($field, false);
+    }
+
+    /**
+     * Returns the value of a field by name
```

Comment:
This needs to explain in more detail what this does.
That it looks at the keys instead of values.
It will not be clear to everyone that it can return null, if the property exists but is set to null.